.github/workflows/appimage.yml

```
@@ -92,6 +93,7 @@ jobs:
           --java-options "-Dcryptomator.ipcSocketPath=\"@{userhome}/.config/Cryptomator/ipc.socket\""
           --java-options "-Dcryptomator.mountPointsDir=\"@{userhome}/.local/share/Cryptomator/mnt\""
           --java-options "-Dcryptomator.showTrayIcon=false"
+          --java-options "-Dcryptomator.integrationsLinux.trayIconsDir=\"${TRAYICONSDIR}\"" \
```

yml context `>` removes any line breaks already.

```suggestion
          --java-options "-Dcryptomator.integrationsLinux.trayIconsDir=\"${TRAYICONSDIR}\""
```